TravelExpenseMain.c

```diff
@@ -0,0 +1,93 @@
+#include<string.h> 
+#include<stdlib.h>
+#include<stdio.h>
+#include<ctype.h>
+ 
+
+// 10 functions
+void getNumber_of_day_onTrip(int); // input the number of day for the trip
+void getDepartureAndArrivalTimes(double , double ); // input the departure time and arrival time
+void getRoundTripAirfare(double , double); // input the round trip Airfare and keep track with total expenses
```

Comment:
Should be like this:

- Naming convention
getNumber_of_day_onTrip -> get_number_of_day_on_trip

- Return type
Note that we can't pass-by-reference in C, so function will need a return type.